proxy/http/HttpConfig.h

```diff
@@ -670,6 +671,7 @@ struct OverridableHttpConfigParams {
   MgmtInt default_buffer_size_index;
   MgmtInt default_buffer_water_mark;
   MgmtInt slow_log_threshold;
+  MgmtInt ssl_client_verify_server;
```

Comment:
More than 255 modes? That seems unlikely. Use MgmtByte unless it's truly an integer.